arwen/host/asyncCall.go

```diff
@@ -407,6 +408,20 @@ func (host *vmHost) createCallbackContractCallInput(
 	}
 
 	if isESDTOnCallBack {
+		/*		contractCallInput.ESDTTokenName = esdtArgs[0]
+				contractCallInput.ESDTValue = big.NewInt(0).SetBytes(esdtArgs[1])
+				contractCallInput.CallValue = big.NewInt(0)
+
+				if functionName == core.BuiltInFunctionESDTNFTTransfer {
+					contractCallInput.ESDTTokenNonce = big.NewInt(0).SetBytes(esdtArgs[1]).Uint64()
+					contractCallInput.ESDTTokenType = uint32(core.NonFungible)
+					contractCallInput.ESDTValue = big.NewInt(0).SetBytes(esdtArgs[2])
+
+					if host.AreInSameShard(contractCallInput.CallerAddr, contractCallInput.RecipientAddr) {
+						contractCallInput.RecipientAddr = contractCallInput.CallerAddr
+					}
+				}
+		*/
```

Comment:
Can the commented code be safely deleted?

Reply:
done